### PR TITLE
Fix issue #1276: remove the localization from package description

### PR DESCRIPTION
**Pull request by xuzhg**
### Issues

*This pull request fixes issue #1276.*

### Description

* Remove the localization from package description

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. </summary>
```
Comment (mikepizzo) [inline src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec]: > only [](start = 115, length = 5)

please remove "only"

File: src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. </summary>
-    <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. Targets .NET 4.5 and .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. Targets .NET 4.5 and .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec]: >only [](start = 120, length = 4)

Please remove "only"

File: src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Release.nuspec [hunk 3]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. </summary>
```
Comment (mikepizzo) [inline src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Release.nuspec]: > only [](start = 115, length = 5)

please remove "only".

File: src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Release.nuspec [hunk 4]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. </summary>
-    <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. Targets .NET 4.5 and .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. Targets .NET 4.5 and .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Release.nuspec]: > only [](start = 119, length = 5)

please remove "only"

File: src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Release.nuspec [hunk 5]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Contains classes and methods that facilitate geography and geometry spatial operations. Supports OData v4 only.</summary>
-    <description>Contains classes and methods that facilitate geography and geometry spatial operations. Support OData v4 only. Targets .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>Contains classes and methods that facilitate geography and geometry spatial operations. Support OData v4 only. Targets .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Release.nuspec]: > Support OData v4 only.  [](start = 104, length = 24)

please remove "Support OData v4 only".  This library sits below OData and is not OData specific.

File: src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Release.nuspec [hunk 6]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Contains classes and methods that facilitate geography and geometry spatial operations. Supports OData v4 only.</summary>
```
Comment (mikepizzo) [inline src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Release.nuspec]: >Supports OData v4 only [](start = 101, length = 22)

please remove "Support OData v4 only".  This library sits below OData and is not OData specific.

File: src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Nightly.Release.nuspec [hunk 7]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Contains classes and methods that facilitate geography and geometry spatial operations. Supports OData v4 only.</summary>
```
Comment (mikepizzo) [inline src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Nightly.Release.nuspec]: > Supports OData v4 only. [](start = 100, length = 24)

please remove "Support OData v4 only".  This library sits below OData and is not OData specific.

File: src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Nightly.Release.nuspec [hunk 8]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Contains classes and methods that facilitate geography and geometry spatial operations. Supports OData v4 only.</summary>
-    <description>Contains classes and methods that facilitate geography and geometry spatial operations. Support OData v4 only. Targets .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>Contains classes and methods that facilitate geography and geometry spatial operations. Support OData v4 only. Targets .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Nightly.Release.nuspec]: > Support OData v4 only.  [](start = 104, length = 24)

please remove "Support OData v4 only".  This library sits below OData and is not OData specific.

File: src/Microsoft.OData.Edm/Build.NuGet/Microsoft.OData.Edm.Nightly.Release.nuspec [hunk 9]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to represent, construct, parse, serialize and validate entity data models. Supports OData v4 only.</summary>
```
Comment (mikepizzo) [inline src/Microsoft.OData.Edm/Build.NuGet/Microsoft.OData.Edm.Nightly.Release.nuspec]: > only [](start = 113, length = 5)

Please remove "only".

File: src/Microsoft.OData.Edm/Build.NuGet/Microsoft.OData.Edm.Nightly.Release.nuspec [hunk 10]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to represent, construct, parse, serialize and validate entity data models. Supports OData v4 only.</summary>
-    <description>Classes to represent, construct, parse, serialize and validate entity data models. Supports OData v4 only. Targets .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>Classes to represent, construct, parse, serialize and validate entity data models. Supports OData v4 only. Targets .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.OData.Edm/Build.NuGet/Microsoft.OData.Edm.Nightly.Release.nuspec]: > only [](start = 117, length = 5)

Please remove "only".

File: src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec [hunk 11]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only.</summary>
```
Comment (mikepizzo) [inline src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec]: > only [](start = 98, length = 5)

Please remove "only".

File: src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec [hunk 12]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only.</summary>
-    <description>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only. Enables construction of OData services and clients. Targets .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only. Enables construction of OData services and clients. Targets .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec]: > only [](start = 102, length = 5)

Please remove "only".

File: src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Release.nuspec [hunk 13]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only.</summary>
```
Comment (mikepizzo) [inline src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Release.nuspec]: > only [](start = 98, length = 5)

Please remove "only".

File: src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Release.nuspec [hunk 14]
```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only.</summary>
-    <description>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only. Enables construction of OData services and clients. Targets .NET Platform Standard 1.1. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
+    <description>Classes to serialize, deserialize and validate OData JSON payloads. Supports OData v4 only. Enables construction of OData services and clients. Targets .NET Platform Standard 1.1.
```
Comment (mikepizzo) [inline src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Release.nuspec]: > only [](start = 102, length = 5)

Please remove "only".

File: src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec (hunk 12 above)
Comment (mikepizzo) [inline src/Microsoft.OData.Core/Build.NuGet/Microsoft.OData.Core.Nightly.Release.nuspec]: > only [](start = 102, length = 5)

Please remove "only".

**Comment by mikepizzo** [comment]
Note that wording doesn't reflect Saurabh's proposed wording (which obsoletes my other comments for Microsoft.OData.Core).  I'm okay with Saurabh's wording, as long as we change "Support for..." to "Documentation for..."

**Comment by xuzhg** [comment]
@mikepizzo Updated the PR. Thanks.

**Pull request merged**